demo-print-file-service/dummy_keys/dummy-key-ssdc-rm-private.asc

```diff
@@ -0,0 +1,109 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
🎨  Would be nice to adjust the config rather than copying these keys in two places, but happy with the path of least resistance in this project.